Makefile

```diff
@@ -36,8 +36,8 @@ unit:
 
 .PHONY: tag
 tag:
-	@ git tag $(VERSION)
+	@ git tag $(VERSION)
```

Comment:
❓

Reply:
My macbook has a bad habit of adding strange characters as spaces that get "executed" as part of the ` git` command, this is just adding a regular space character back.